data/nation/coutense.ruleset

```diff
@@ -0,0 +1,39 @@
+[nation_coutense]
+
+name   = _("Coutense")
+plural = _("?plural:Coutenses")
+groups = "Medieval", "European"
+
+legend = _("Couto Misto was a microstate on the border between Portugal\
+ and Spain. It consisted of three villages ruled by a republic, where\
+ the heads of families voted for a 'juiz.' It was formed in the 10th\
+ century and annexed by Portugal and Spain in 1868 under the Treaty of\
+ Lisbon. In the 1990s, interest in recovering Couto Misto emerged.")
+
+leaders = {
+ "name",                   "sex"
+ "Delfim Modesto Brandão", "Male"
+}
```

Comment:
We need more than just one leader, here is some suggestions:

[María do Ceo](https://gl.wikipedia.org/wiki/Mar%C3%ADa_do_Ceo) Sang the anthem to Couto Mixto.
[Luis García Maña](https://gl.wikipedia.org/wiki/Lu%C3%ADs_Garc%C3%ADa_Ma%C3%B1%C3%A1) One of the modern honorary judges,

Reply:
Also Aldara de Celanova, and [Rudesind Guterri](https://en.wikipedia.org/wiki/Rudesind)